packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -268,7 +263,7 @@ export const DownloadCartPage: React.FunctionComponent<
                   </div>
                 </div>
               </div>
-              <div className="availableForDownloadTableContainer container bootstrap-4-backport">
+              <div className="availableForDownloadTableContainer container">
```

Comment:
Remove bootstrap-4-backport from the parent -- it breaks the MUI TextField styling